embedded/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["Riccardo Casatta <riccardo@casatta.it>", "Dev Random <c1.devrandom@niftybox.net>"]
+edition = "2018"
+readme = "README.md"
+name = "embedded"
+version = "0.1.0"
+
+[dependencies]
+cortex-m = "0.6.0"
+cortex-m-rt = "0.6.10"
+cortex-m-semihosting = "0.3.3"
+panic-halt = "0.2.0"
+alloc-cortex-m = "0.4.1"
+bitcoin = { path="../", default-features = false, features = ["no-std", "secp-lowmemory"] }
+
+[[bin]]
+name = "embedded"
+test = false
+bench = false
+
+[profile.release]
+codegen-units = 1 # better optimizations
+debug = true # symbols are nice and they don't increase the size on Flash
```

Comment:
the embedded dir is for testing on qemu (see the github workflow changes) on a specific device, so that we can tell if std is somehow pulled in by mistake (std crate doesn't exist for the device).  the only reason to optimize here is to ensure we don't exceed the flash size on the emulated device.

In other words, this is not something we expose to the developer that is using this crate, so I don't think it's a problem.